src/App/Components/PasswordReset/ForgotPassword.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import axiosInstance from '../../../configs/axios';
+import Alert from '../Common/Alert';
+
+export class PasswordReset extends Component {
+  state = { email: '' };
```

Comment:
@nkpremices any reason why you're not doing this in a constructor?

Reply:
Because I'm not using redux @MemunaHaruna

Reply:
But if you think it is not a good practice, I'm going to put it in a constructor.

Reply:
My next question is, why are you not using Redux? I'm not saying whether it's a good practice or not. I want to understand your reasons for using this method.

Reply:
Oh.. Now I understand the question.
I know that only redux requires to initialize the state in the constructor @MemunaHaruna

Reply:
Actually, a component's state (usually initialised in the constructor) can be totally separate from redux store. Having a default state in the constructor is just a regular JS/React pattern and is not tightly coupled to Redux at all.

Reply:
Understood @MemunaHaruna. I actually didn't know that.
Let me implement that